recipes/openjdk/build.sh

```diff
@@ -0,0 +1,20 @@
+#/bin/bash -eu
+
+if [[ `uname` == "Linux" ]]
+then
+    mv lib/amd64/jli/*.so lib
+    mv lib/amd64/*.so lib
+    rm -r lib/amd64
+fi
+
+cp -r bin $PREFIX
+cp -r include $PREFIX
+cp -r jre $PREFIX
+cp -r lib $PREFIX
+
+for action in activate deactivate
+do
+    dir=$PREFIX/etc/conda/$action.d
+    mkdir -p $dir
+    cp $RECIPE_DIR/scripts/$action.sh $dir/java_home.sh
```

Comment:
Yeah... also one other argument to keep this as java_home - there should only ever be one such script.  If more than one such script exists, then the actual value of JAVA_HOME gets pretty undependable - depends on script naming for order of resolution.